### PR TITLE
docs(api): Update labware section of API docs for labware v2 names

### LINKED ISSUE #918: Mistake in documentation for container: tube-rack-15_50ml
## overview

The documentation here http://docs.opentrons.com/containers.html for tube-rack-15_50ml appears to be incorrect. The indexing of locations shown in the image conflicts with the text.

### LINKED ISSUE #2050: Documentation: Include brand/model of default container definitions
## overview

As a user it would be really helpful to know exactly which brands and models the default container definitions are based on. Then we could say "Hey, this isn't the brand of PCR plate I'm using. I should probably double check the dimensions".

This would help us diagnose issues that were invisible on the OT-One due to bottom-of-well calibrations.

### LINKED ISSUE #2051: Documentation: Update the container definition JSON location in the docs
## overview

Currently the default container visualization is being loaded from a JSON in the `master` branch last updated over a year ago. As a user, I would like to see an up-to-date list from the `edge` branch.

[Current docs](https://docs.opentrons.com/labware.html)

The link in question appears about a third of the way down the page.

### LINKED ISSUE #3615: V2 Labware: Update API docs with new usage and LL
## overview

Prior to the launch of the v2 labware definitions, https://docs.opentrons.com/labware.html needs to be updated to:

- Link to the labware library
- Say something about the usage of legacy labware load names 
- Correct any outstanding mistakes in the labware docs (e.g. #918)

### PR DESCRIPTION
**Pull request by mcous**
## overview

This PR updates the Labware section of the API docs to reference the upcoming Labware Library and note the deprecation of the old load names.

- Closes #3615
- Closes #2050
- Closes #2051
- Closes #918

**This PR should not be merged until we are ready to release `3.10.0`**

## changelog

- Update labware section of API docs for labware v2 names

## review requests

Please check docs for correctness

### CODE REVIEW

File: api/docs/source/labware.rst [hunk 1]
```diff
@@ -450,83 +67,70 @@ Labware Module
...
-that was already created and calibrated.
+Create
+======
 
-**Note** There is some specialty labware that will require you to specify the type within your labware name.
-If you are creating a custom tiprack, it must be `tiprack`-REST-OF-LABWARE-NAME in order for the program to act reliably.
+.. Note::
+    The current custom labware creation mechanisms in the API are fairly
+    limited. We're working on a much more robust system for custom labware
+    defintions. If the current API isn't able to support your labware, please
```
Comment (IanLondon) [inline api/docs/source/labware.rst]: defintions -> definitions

File: api/docs/source/labware.rst [hunk 2]
```diff
@@ -450,83 +67,70 @@ Labware Module
...
 .. code-block:: python
 
-    from opentrons.data_storage import database
-    database.delete_container('3x6_plate')
+    custom_plate_name = 'custom_18_wellplate_200ul'
 
-This allows you to make changes to the labware within the database under the same name.
+    if plate_name not in labware.list():
+        labware.create(
+            plate_name,        # name of you labware
```
Comment (IanLondon) [inline api/docs/source/labware.rst]: plate_name -> custom_plate_name right?

File: api/docs/source/labware.rst [hunk 3]
```diff
@@ -549,14 +153,46 @@ will print out...
...
+
+    database.delete_container('custom_18_wellplate_200ul')
+
+.. Note::
+    There is some specialty labware that will require you to specify the
+    type within your labware name. If you are creating a custom tip rack, it
+    must be ``tiprack-REST-OF-LABWARE-NAME`` in order for the software to act
+    reliably.
+
+List (depreacted)
```
Comment (IanLondon) [inline api/docs/source/labware.rst]: depreacted -> deprecated

File: api/docs/source/labware.rst [hunk 4]
```diff
@@ -849,6 +500,108 @@ will print out...
...
+
+.. Note::
+    If your labware is missing from the list above, or you're unsure how to
+    update your protocol's load names, please contact our support team
+
+The following load names do not have a new definitions available, and could
+eventually be removed. They will continue to function normally for now. If you
+have any concerns about their deprecation and/or removal, please reach out!
+
+- ``24-vial-rack``
```
Comment (IanLondon) [inline api/docs/source/labware.rst]: PCR-strip-tall & trash-box should have equivalents

File: api/docs/source/labware.rst [hunk 5]
```diff
@@ -579,33 +218,51 @@ The OT-One deck and labware are all set up with the same coordinate system - let
...
+Once a labware is loaded into your protocol, you can easily access the many
+wells within it using ``wells()`` method. ``wells()`` takes the name of the
+well as an argument, and will return the well at that location.
 
 .. code-block:: python
 
-    plate.wells('A1')
-    plate.wells('H12')
+    a1 = plate.wells('A1')
+    h12 = plate.wells('D6')
```
Comment (Laura-Danielle) [inline api/docs/source/labware.rst]: Confused as to why we called this 'D6' instead. If we are keeping this well, then we should rename the var to `d6` to match the style of code snippet above.

File: api/docs/source/labware.rst [hunk 6]
```diff
@@ -579,33 +218,51 @@ The OT-One deck and labware are all set up with the same coordinate system - let
...
+also labelled on standard labware. In the API, rows are given letter names
+(``'A'`` through ``'D'`` for example) and go left to right, while columns are
+given numbered names (``'1'`` through ``'6'`` for example) and go from front to
+back.
+
+You can access a specific row or column by using the ``rows()`` and
+``columns()`` methods on a labware. These will return all wells within that row
+or column.
 
 .. code-block:: python
```
Comment (Laura-Danielle) [inline api/docs/source/labware.rst]: This section is a bit confusing if we don't actually load a 96 well plate.

File: api/docs/source/labware.rst (hunk 5 above)
Comment (mcous) [reply api/docs/source/labware.rst]: Oops, dumb typo

**Comment by IanLondon** [comment]
Dunno if it's in or out of scope for this ticket, but there are leftover `generic_96_wellplate_*`'s in `new_protocol_api.rst`

File: api/docs/source/labware.rst [hunk 7]
```diff
@@ -849,6 +500,111 @@ will print out...
...
+have any concerns about their deprecation and/or removal, please reach out!
+
+- ``24-vial-rack``
+- ``48-vial-plate``
+- ``5ml-3x4``
+- ``96-well-plate-20mm``
+- ``MALDI-plate``
+- ``T25-flask``
+- ``T75-flask``
+- ``biotix-resevoir``
```
Comment (IanLondon) [inline api/docs/source/labware.rst]: did this exist as standard labware? If so `biotix-resevoir` needs to be added to LW_NO_EQUIVALENT list in back_compat.py

But I don't see it in `labware.list()`

File: api/docs/source/labware.rst [hunk 8]
```diff
@@ -849,6 +500,111 @@ will print out...
...
+| ``tiprack-200ul``                             | ``tipone_96_tiprack_200ul``                                                                 | If possible, please use an Opentrons tip rack rather than a rack with a slot adapter |
++-----------------------------------------------+---------------------------------------------------------------------------------------------+--------------------------------------------------------------------------------------+
+| ``tiprack-1000ul``                            | ``opentrons_96_tiprack_1000ul``                                                             | If possible, please use an Opentrons tip rack rather than a rack with a slot adapter |
++-----------------------------------------------+---------------------------------------------------------------------------------------------+--------------------------------------------------------------------------------------+
+| ``trash-box`                                  | ``agilent_1_reservoir_290ml``                                                               |                                                                                      |
```
Comment (IanLondon) [inline api/docs/source/labware.rst]: This is another that is a distant mapping, might merit a note, eg "The trash-box definition is no longer supported, we recommend using a 1-well trough as a liquid trash" or something

File: api/docs/source/labware.rst (hunk 7 above)
Comment (mcous) [reply api/docs/source/labware.rst]: Shoot, I meant to remove this. @Laura-Danielle pointed it out to me in LW_NO_EQUIVALENT. Looks like it's a holdover on my own machine from some customer protocol testing

**Pull request merged**